### DISCUSSION TITLE
Group 9: China SOE Reform

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#165] [Group Projects Feedback]
Group 9, please add a short title so we can refer to your project by name! Macro and business reforms are a great area of interest, so let me provide some high level feedback on the design here so you can refine it moving forward.

You write: "In the study, those enterprises whose state ownership in total top 10 ownership shifts from above 50% to below 50% are regarded as the treatment group". Unfortunately, I have no idea what this means. Is it to say that the treatment group is "any company where the state moves from more than 50% to less than 50%", or is it more nuanced than that? For example, under that definition, the "treatment group" could have a company that goes from 51% to 49% state ownership, while the control group has a company that goes from 100% state ownership to 51%. Is that right? As a suggestion -- you might instead consider "continuous" treatment, in which exposure is "state ownership in given year" divided by state ownership in 2013 -- effectively an index. So, narrowing down on your treatment is important -- as are interaction-type considerations such as the baseline level of state ownership.

Of course, the design has endogeneity challenges -- there must be a _reason_ the state reduces its ownership in specific companies. Is it because they are losing money, for example? If that's the case, your set of controls should be based around building up a comparison between companies that are "similar" other than their change in state ownership levels. Thankfully you have lots of financial data it seems to set up these micro-comparator groups (identification via "strata", coming later in the term), so think very very carefully about how to do it in such a way that the state's choice to divest would not be driven by the excluded financials. Financial data in this case is challenging because you must have a strong theory of why you are not running a "kitchen sink" regression (that is, just listing all the "stuff" you have as controls) because such models quickly create statistical problems when the controls include channels and colliders, as discussed in Béa's lecture.

The data seems to have a substantial gap in calculating ACAR, which you suggest is your main outcome even though ... there is no data available on it? If you are going to use derived data as an outcome, you _must_ define it! Think carefully about what your theory of change suggests a "more private" company would try to achieve; and about what changes in outcomes would be statistical artifacts (ie, are there tax changes/ state remission/ regulatory treatments/etc). You have access to lots of data points that could be inputs or outcomes -- do not be afraid to have multiple outcomes! (ie, a private company could decrease investments while increasing profits). Also -- forget about timeline, and definitely don't worry about processing power considerations. Research of this kind (purely secondary) generally takes years and doesn't really need that kind of strictness since you are not funding anything; that is more for fieldwork when you have salaries, surveys, and other practical considerations. "Desk" research is funded by your "salary" (and we have given you an effectively unlimited research budget) so you don't need to clock it out in the same way for practical purposes (unlike groups who want to run interventions and surveys across large countries).

In sum -- this is a great project idea and just needs some refinement on the definitions of the treatment groups and the outcomes. Excited to see where you take it!

**Comment by BeaLeydier** [comment]
Hi team

Love a macro topic on the economy as a whole! This is a challenging but v interesting design.

My main comments building on Ben's comments are below 

1. Can you expand on what the 2015 policies were, and whether all companies were eligible to them? It's not very clear if some companies didn't see their share composition change because they weren't eligible/targeted, or if because their share composition to start with was such that the reform didn't have any impact on them (for example, they were already semi privatized prior to the reform). You need to clarify if you are trying to measure the impact of the policy that was implemented (and if yes specify more what exactly the policy consisted of), or if you are tying to measure the impact of a certain % of state ownerhsip in a company in general, which I am understanding could be a result of that policy change, and as Ben said may be better studied in a continuous fashion. This is important, because you don't want to be comparing companies that aren't comparable - eg. companies that were already privatized are probably not comparable in 2015 to companies that weren't already privatized by then.

2. Your main outcome (ACAR) is at the company level -- it could be interesting also looking at outcomes at the market-level, i.e. are there any equilibirum effects on other companies or the economy as a whole. How will this affect other, non SOEs companies? or SOEs companies that are not included in the reform (if there are some)? 

3. Why is ACAR your main outcome? What is the unit and scope of it? What is your prediction of how the reform will affect it? Why is it relevant for measuring the effect of the reform (and/or of state ownerhsip in general)? Why is it an indicator we should care about, what does it tell us about a company or an economy, and how is it relevant to other companies/economies/contexts?

Cheers!

**Comment by bbdaniels** [comment]
# Concept Note Review

You continue to demonstrate an excellent understanding of the context and an interest in a ver challenging puzzle. The main problem here is that you have not increased the specificity of the design at all from the pitch -- most of the notes here say "we will figure it out when we have the data" which is not really a complete design for this assignment. You need to have developed specific comparison groups and outcome measures for the final submission, so that you can present detailed numbers about the comparisons you would make when you have access to actual data. Without that, you really only have the project pitch.

You need at this stage to get concrete about the data. It is OK that you do not actually have access to it in this case, but you need to come up with plausible values and patterns of all the elements you are interested in so you can simulate the experiment and discuss things like the detectable sizes of impacts etc, given the types of changes in state ownership that you anticipate in the sample. Do not worry about the "technical" challenges you have outlined in Complications -- we are assuming you have effectively infinite computing resources etc. What we need to understand is the fundamental measurement, design, and comparator groups you will use.

In particular, I do not understand the key sentence "the proportion of the company's state ownership in the total top 10 ownership in that year divided by the status in 2013". This seems to define your entire treatment, but you have given no examples of what it means for various companies, how many companies there are, how many would be in treatment and control, and what are the underlying reasons for changes in state ownership over that period. Therefore the context for understanding the measures -- and interpreting your results. -- is lacking, as is understanding the power of the project based on an understanding of the correlations within and across the expected treatment and control groups. Focus on nailing this down in very concrete terms so you can very clearly express the regression model and the exact definitions of your data in the final submission, and so you can simulate those concretely. Keep it up, you are on the way!
